source/index.txt

```diff
@@ -1,10 +1,91 @@
-==================
-Kotlin Sync Driver
-==================
+==========================
+{+driver-long+}
+==========================
 
-.. default-domain:: mongodb
+.. toctree::
+   :titlesonly:
+   :maxdepth: 1
 
-Your words here. Don't forget to add a toctree!
+   /faq
+   /connection-troubleshooting
+   /issues-and-help
+   /compatibility
+   View the Source <https://github.com/mongodb/mongo-java-driver/tree/master/driver-kotlin-sync>
 
-Have a lovely day!
+Introduction
+------------
 
+Welcome to the documentation site for the {+driver-long+}, the official
+MongoDB driver for synchronous Kotlin applications. Download the driver by using
+`Maven <https://maven.apache.org/>`__ or `Gradle <https://gradle.org/>`__, or set up a
+runnable project by following our Quick Start guide.
+
+.. tip:: Other Kotlin Platforms for MongoDB
+
+   If your Kotlin application requires asynchronous processing, use the
+   :driver:`Coroutine Driver </kotlin/coroutine/current>`, which uses coroutines for
+   server-side applications.
+   
+   If you are developing an Android or Kotlin Multiplatform (KMP)
+   application, you can use the :realm:`MongoDB Atlas Device Kotlin SDK </sdk/kotlin/>` 
+   to implement Device Sync and to manage your Realm data.
+
+Quick Start
+-----------
+
+Learn how to establish a connection to MongoDB Atlas and begin
+working with data in the :ref:`Quick Start <kotlin-sync-quick-start>` section.
+
+What's New
+----------
+
+For a list of new features and changes in each version, see the :ref:`What's New <kotlin-sync-whats-new>`
+section.
+
+Quick Reference
+---------------
+
+See driver syntax examples for common MongoDB commands in the
+:ref:`Quick Reference <kotlin-sync-quick-reference>` section.
+
+FAQ
+---
+
+For answers to commonly asked questions about the {+driver-short+}, see the
+:ref:`FAQ <kotlin-sync-faq>` section.
+
+Connection Troubleshooting
+--------------------------
+
+For solutions to some issues you might experience when connecting to a MongoDB
+deployment while using the {+driver-short+}, see the 
+:ref:`Connection Troubleshooting <kotlin-sync-connection-troubleshooting>` section.
+
+Issues & Help
+-------------
+
+Learn how to report bugs, contribute to the driver, and find more resources for
+asking questions and receiving help in the :ref:`Issues & Help <kotlin-sync-issues-and-help>` section.
+
+Compatibility
+-------------
+
+For the compatibility charts that show the recommended {+driver-short+} version for each
+MongoDB Server version, see the :ref:`Compatibility <kotlin-sync-compatibility>` section.
+
+Learn
+------
+
+Visit the Developer Hub to learn more about the {+driver-long+}.
+
+Developer Hub
+~~~~~~~~~~~~~
+
+The Developer Hub provides tutorials and social engagement for
+developers.
+
+To learn how to use MongoDB features with the {+driver-short+}, see the
+`How To's and Articles page <https://www.mongodb.com/developer/languages/kotlin/>`__.
```

Comment:
Nit: Change How To's to How-Tos: [style guide](https://www.mongodb.com/docs/meta/style-guide/terminology/alphabetical-terms/#h) and [outside source](https://www.dictionary.com/browse/how-to). I also don't see this title on the link you've sent, is "how tos and articles" the way that we normally refer to this page?

Reply:
Good shout! Will fix